box.py

```diff
@@ -0,0 +1,46 @@
+import numpy as np
+
+class Box:
+    def __init__(self, box_dim):
+        self.box_dim = box_dim
+        
+    def wrap(self, coordinates):
+        """Wraps the coordinates within the box dimensions
+
+        Parameters
+        ----------
+        box_dim : np.array
+            Dimensions of the box in reduced units.
+        coordinates : np.array
+            Array of the atomic coordinates.
+
+        Returns
+        -------
+        coordinates : np.array
+            Arrays of the wrapped atomic coordinates.
+        """
+        if len(coordinates.shape) == 1:
+            coordinates -= self.box_dim * np.round(coordinates / self.box_dim)
+        else:
+            coordinates -= self.box_dim * np.round(coordinates / self.box_dim[np.newaxis, :])
+        return coordinates
+        
+    def minimum_image_distance(self,coord_i, coord_j):
```

Comment:
It would be nice to allow for `coord_j` to be able to be a numpy array of multiple coordinates to get multiple minimum images
at once.

Reply:
Definitely possible and something I was thinking too. I was talking with Van and Joel to see what my output should be since they are going to be using the minimum image distance for computing the single particle and pairwise energies. I'll convene with them again.